ruby-gem/lib/calabash-android/helpers.rb

```diff
@@ -92,7 +92,11 @@ def checksum(file_path)
 end
 
 def test_server_path(apk_file_path)
-  "test_servers/#{checksum(apk_file_path)}_#{Calabash::Android::VERSION}.apk"
+  if ENV['TEST_APP_PATH']
+    ENV['TEST_APP_PATH']
```

Comment:
With this we can supply prebuilt Calabash servers and don't built anything at all